tests/test_request_validator.py

```diff
@@ -0,0 +1,15 @@
+from async_customerio import validate_signature
+
+
+def test_validate_signature():
```

Comment:
Please add a test for a negative scenario.

Reply:
added more test cases